<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsDialog</class>
 <widget class="QDialog" name="StatsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupUser">
     <property name="title">
      <string>User statistics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelAlltimeULText">
        <property name="text">
         <string>All-time upload:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelAlltimeUL">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelAlltimeDLText">
        <property name="text">
         <string>All-time download:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelAlltimeDL">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelGlobalRatioText">
        <property name="text">
         <string>All-time share ratio:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelGlobalRatio">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelWasteText">
        <property name="text">
         <string>Session waste:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelWaste">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelPeersText">
        <property name="text">
         <string>Connected peers:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelPeers">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCache">
     <property name="title">
      <string>Cache statistics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCacheHitsText">
        <property name="text">
         <string>Read cache hits:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelCacheHits">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelTotalBufText">
        <property name="text">
         <string>Total buffer size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelTotalBuf">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupPerf">
     <property name="title">
      <string>Performance statistics</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelWriteStarveText">
        <property name="text">
         <string>Write cache overload:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelWriteStarve">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelReadStarveText">
        <property name="text">
         <string>Read cache overload:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelReadStarve">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelQueuedJobsText">
        <property name="text">
         <string>Queued I/O jobs:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelQueuedJobs">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelJobsTimeText">
        <property name="text">
         <string>Average time in queue:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelJobsTime">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelQueuedBytesText">
        <property name="text">
         <string>Total queued size:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QLabel" name="labelQueuedBytes">
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
